### PR TITLE
refactor(limits): refine buffer logic and default values

### PR DESCRIPTION
**Pull request by dohaki**
Refine default limits buffer multipliers and logic

### CODE REVIEW

File: api/_constants.ts [hunk 1]
```diff
@@ -178,9 +178,12 @@ export const DEFAULT_SIMULATED_RECIPIENT_ADDRESS =
   "0xBb23Cd0210F878Ea4CcA50e9dC307fb0Ed65Cf6B";
 
 export const DEFAULT_LIMITS_BUFFER_MULTIPLIERS = {
-  recommendedDepositInstant: "0.1",
-  depositInstant: "0.2",
-  depositShortDelay: "0.2",
+  [TOKEN_SYMBOLS_MAP.DAI.symbol]: "0.8",
```
Comment (nicholaspai) [inline api/_constants.ts:181]: Nit: I think in general we shouldn't be hardcoding override values in the code and instead should rely on env vars but this is fine for now

File: api/_utils.ts [hunk 2]
```diff
@@ -1481,76 +1481,14 @@ export function isSwapRouteEnabled({
...
-    DEFAULT_LIMITS_BUFFER_MULTIPLIERS.recommendedDepositInstant
+export function getLimitsBufferMultiplier(symbol: string) {
+  const limitsBufferMultipliers: Record<string, string> = process.env
+    .LIMITS_BUFFER_MULTIPLIERS
+    ? JSON.parse(process.env.LIMITS_BUFFER_MULTIPLIERS)
+    : {};
+  return ethers.utils.parseEther(
+    limitsBufferMultipliers[symbol] ||
+      DEFAULT_LIMITS_BUFFER_MULTIPLIERS[symbol] ||
+      "1"
```
Comment (nicholaspai) [inline api/_utils.ts:1492]: By removing the `DEFAULT_LIMITS_BUFFER_MULTIPLIERS` constant as per my other comment we could shorten this a bit. Basically, either the buffer is specified in an env var or we default to 1, no intermediary hardcoded buffer

File: api/_utils.ts [hunk 3]
```diff
@@ -1481,76 +1481,14 @@ export function isSwapRouteEnabled({
...
- * @param symbol The token symbol to retrieve the multiplier for.
- * @param fromChainId The chain ID of the origin chain.
- * @param toChainId The chain ID of the destination chain.
- */
-export const getRecommendedDepositInstantMultiplier =
-  makeLimitsBufferMultiplierGetter(
-    "RECOMMENDED_DEPOSIT_INSTANT_MULTIPLIER",
-    DEFAULT_LIMITS_BUFFER_MULTIPLIERS.recommendedDepositInstant
+export function getLimitsBufferMultiplier(symbol: string) {
+  const limitsBufferMultipliers: Record<string, string> = process.env
```
Comment (nicholaspai) [inline api/_utils.ts:1485]: nit: can we cap the buffer to 100%? Not sure why we'd want to ever show more than 100%

**Review by nicholaspai — APPROVED** [review]
Looks great, left some nits

File: src/views/Bridge/components/BridgeForm.tsx [hunk 4]
```diff
@@ -148,7 +150,7 @@ const BridgeForm = ({
           errorText={validationErrorTextMap[validationError].replace(
             "[MAX_DEPOSIT_SHORT_DELAY]",
```
Comment (nicholaspai) [inline src/views/Bridge/components/BridgeForm.tsx:151]: should this be just MAX_DEPOSIT..?

File: api/_constants.ts (hunk 1 above)
Comment (dohaki) [reply api/_constants.ts:181]: ok removed

File: api/_utils.ts (hunk 3 above)
Comment (dohaki) [reply api/_utils.ts:1484]: yea makes sense! https://github.com/across-protocol/frontend-v3/pull/1080/commits/e161390500014181b6cea17638522b22a1d01df5

File: src/views/Bridge/components/BridgeForm.tsx (hunk 4 above)
Comment (dohaki) [reply src/views/Bridge/components/BridgeForm.tsx]: https://github.com/across-protocol/frontend-v3/pull/1080/commits/e161390500014181b6cea17638522b22a1d01df5

**Pull request merged**